core/src/main/java/nl/nn/adapterframework/doc/model/ElementChild.java

```diff
@@ -0,0 +1,116 @@
+/* 
+Copyright 2020, 2021 WeAreFrank! 
+
+Licensed under the Apache License, Version 2.0 (the "License"); 
+you may not use this file except in compliance with the License. 
+You may obtain a copy of the License at 
+
+    http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software 
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+
+package nl.nn.adapterframework.doc.model;
+
+import java.util.function.Predicate;
+
+import org.apache.logging.log4j.Logger;
+
+import lombok.Getter;
+import lombok.Setter;
+import nl.nn.adapterframework.doc.DocWriterNew;
+import nl.nn.adapterframework.util.LogUtil;
+
+/**
+ * Base class of FrankAttribute and ConfigChild. This class was introduced
+ * to implement the following common logic:
+ * <ul>
+ * <li> The decision whether to include an attribute or config child in the XML schema
+ * is based on the same information.
+ * <li> The structure is very similar in the XML schema for config children and
+ * attributes. In both cases, we have cumulative groups that include inherited
+ * items and declared groups that hold only items at the present level of the
+ * inheritance hierarchy. Please see this in action at {@link DocWriterNew}.
```

Comment:
Vergeet niet je HTML tags te sluiten!

Reply:
Gedaan. Ik heb op &lt;ul&gt; en &lt;ol&gt; gezocht om te zien of dit probleem ook elders speelt. Ik heb dit probleem nergens anders gevonden.

Reply:
De li's moeten ook nog gesloten worden

Reply:
Ik heb een pagina gevonden waar staat dat dat niet hoeft. Zie https://stackoverflow.com/questions/33787753/javadoc-closing-tags